docker/cygnus-ngsi/cygnus-entrypoint.sh

```diff
@@ -596,6 +596,9 @@ elif [ "$CYGNUS_POSTGRESQL_HOST" != "" ]; then
     sed -i '/'${CYGNUS_AGENT_NAME}'.sinks.postgresql-sink.postgresql_username/c '${CYGNUS_AGENT_NAME}'.sinks.postgresql-sink.postgresql_username = '${CYGNUS_POSTGRESQL_USER} ${FLUME_HOME}/conf/${AGENT_CONF_FILE}
     sed -i '/'${CYGNUS_AGENT_NAME}'.sinks.postgresql-sink.postgresql_password/c '${CYGNUS_AGENT_NAME}'.sinks.postgresql-sink.postgresql_password = '${CYGNUS_POSTGRESQL_PASS} ${FLUME_HOME}/conf/${AGENT_CONF_FILE}
     # The following are optional and disabled by default
+    if [ "$CYGNUS_POSTGRESQL_DATABASE" != "" ]; then
```

Comment:
Maybe the new env var could be mentioned in the CHANGES_NEXT_RELEASE file. Something like:

```
- Fix: new env var CYGNUS_POSTGRESQL_DATABASE to set Postgresql database in Docker
```

Reply:
In addition, maybe the new env var should be included in doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md documentation? (Not fully sure...)

Reply:
I agree, It should be included in the install_with_docker doc.

Reply:
Fixed in b475f0e